functional-tests/src/main/java/com/redhat/che/selenium/core/workspace/RhCheTestWorkspaceProvider.java

```diff
@@ -11,53 +11,27 @@
  */
 package com.redhat.che.selenium.core.workspace;
 
-import com.google.common.util.concurrent.ThreadFactoryBuilder;
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
 import com.redhat.che.selenium.core.client.RhCheTestWorkspaceServiceClient;
-import java.util.concurrent.ArrayBlockingQueue;
-import java.util.concurrent.Executors;
-import java.util.concurrent.TimeUnit;
-import javax.inject.Named;
-import org.eclipse.che.commons.lang.concurrent.LoggingUncaughtExceptionHandler;
-import org.eclipse.che.selenium.core.client.TestWorkspaceServiceClientFactory;
-import org.eclipse.che.selenium.core.user.DefaultTestUser;
 import org.eclipse.che.selenium.core.user.TestUser;
-import org.eclipse.che.selenium.core.utils.WorkspaceDtoDeserializer;
-import org.eclipse.che.selenium.core.workspace.AbstractTestWorkspaceProvider;
 import org.eclipse.che.selenium.core.workspace.TestWorkspace;
+import org.eclipse.che.selenium.core.workspace.TestWorkspaceProvider;
+import org.eclipse.che.selenium.core.workspace.WorkspaceTemplate;
 
 @Singleton
-public class RhCheTestWorkspaceProvider extends AbstractTestWorkspaceProvider {
```

Comment:
@rhopp @Katka92 @ScrewTSW Please, take a look whether this is an OK fix after upstream changes in functional tests framework.

Reply:
From the looks of it seems OK. Let PR check job do the "heavy lifting" :-D If PR check job passes, it should be ok.

Reply:
Doesn't look like it passes :-(

Reply:
But we will have to wait for a fix [upstream](https://github.com/eclipse/che/pull/11402) plus an extra night for it to be populated to the maven repo.

Reply:
yes, PR https://github.com/eclipse/che/pull/11402 replaces the one mentioned in [the description](https://github.com/redhat-developer/rh-che/pull/931#issue-218435467): https://github.com/eclipse/che/pull/11370 that is now obsolete.